### PR TITLE
Add ppt/pptx support

### PR DESCRIPTION
**Pull request by DevChoganwala**
Fixes issue #2 
ppt/pptx conversion by:
- Converting ppt/pptx files to pdf using `unoconv`
- Converting obtained pdf file to txt using `pdftotext`

### CODE REVIEW

File: src/SearchEverywhere.py [hunk 1]
```diff
@@ -136,6 +136,97 @@ def doc_to_txt(filename):
...
+            os.chdir("../")
+        else:
+            print("Error! No such file exists!")
+            sys.exit()
+    else:
+        files_counter = 0
+        for nameOfFile in os.listdir():
+            if nameOfFile.endswith(".ppt"):
+                files_counter += 1
+                # convert odp to pdf
```
Comment (suraj-singh12) [inline src/SearchEverywhere.py:162]: change the comment to 
convert ppt to pdf

File: src/SearchEverywhere.py [hunk 2]
```diff
@@ -260,14 +351,14 @@ def main():
             text("*.txt",keyword,os.path.abspath("converted"))
             sys.exit()
         
-        # * means files of all types (txt, pdf, odt, odp, docx, doc)
+        # * means files of all types (txt, pdf, odt, odp, docx, doc, ppt, pptx)
         odp_dir=''
         odt_dir=''
         pdf_dir=''
         docx_dir=''
         doc_dir=''
```
Comment (suraj-singh12) [inline src/SearchEverywhere.py:359]: add these lines below this
ppt_dir = ''
pptx_dir = ''

**Review by suraj-singh12 — CHANGES_REQUESTED** [review]
line 418, comparing ppt_dir with ''
line 425, comparing pptx_dir with ''
it's ok, but, ensure you add these after line 359:
ppt_dir = ''
pptx_dir = ''
 
only then it is justifiable why we checking ppt_dir != '' and pptx_dir != '' at line 418, and line 425
otherwise the program might fail, when we search another format, because variable ppt_dir, pptx_dir are not defined 
so how can we compare at 418 and 425

**Comment by DevChoganwala** [comment]
Understood, I guess I missed those details, now that i am checking, rm -rf never got executed and i had unremoved converted-ppt/pptx folders. Will make these changes 👍

**Review by suraj-singh12 — APPROVED** [review]
good work man !!

**Pull request merged**